core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -194,6 +194,24 @@ span.additional-metric-title {
   color: #777;
 }
 
+#additional-metrics {
+  float: left;
+  padding-bottom: 10px;
+}
+
+#links {
+  float: left;
+  padding-left: 100px;
```

Comment:
Calling this "absolute layout" seems misleading, given what `absolute` usually means in this context (e.g. `position: absolute`). These links are positioned "relatively", to the right of the "additional metrics" `<div>`, with a fixed amount of padding.

I'm also not sure what you mean by "inline layout like the rest of this"; again, "inline" has a specific meaning in CSS, e.g. `display: inline`, that I don't think anything else on the page is using. Everything else is basically structured as `<div>`s and `<table>`s that are `display: block` by default meaning that they don't fall next to each other on the same line but each get their own lines, hence the horizontal wasted space.

I'm open to trying something like a `float: right` for the links, though relative to this that would mostly serve to make them farther away from the LHS where the user's eyes and mouse are likely to start out.

Here's a screen fwiw:

![](https://s3.amazonaws.com/f.cl.ly/items/3m3X3B1l0a2t0D1O1n1y/Screen%20Shot%202015-03-21%20at%207.53.25%20PM.png)

Lmk what you think.